debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -318,7 +318,7 @@ protected StructGenerator createValueGenerator(Schema schema, TableId tableId, L
         for (int i = 0; i < columns.size(); i++) {
             Column column = columns.get(i);
 
-            ValueConverter converter = createValueConverterFor(tableId, column, schema.field(column.name()));
+            ValueConverter converter = createValueConverterFor(tableId, column, schema.field(fieldNamer.fieldNameFor(column)));
```

Comment:
I think it's good, but can we add a test for this? It's a very central location, so better safe than sorry.